src/pages/resultsView/cancerSummary/CancerSummaryChart.tsx

```diff
@@ -620,6 +625,29 @@ export class CancerSummaryChart extends React.Component<
         });
     }
 
+    @autobind
+    private getData(dataType: DataType) {
+        return tsvFormat(this.convertDataToDownloadData(this.props.data));
+    }
+
+    @autobind
+    private convertDataToDownloadData(data: any[][]): any[] {
```

Comment:
Try to avoid using 'any' unless really necessary.  data arg comes from this.props.data which is typed.  You may need to extract that.

Reply:
updated to using typed data